### ISSUE TITLE
Supporting for detection of `link:not([href]):not([imagesrcset])`

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#717]
`link` element should have `href` attribute or `imagesrcset` attribute.

# Examples 

## Correct

```html
<link rel="stylesheet" href="./style.css">
```

## Incorrect

```html
<link rel="stylesheet">
```

# Reference

Nu Html Checker warns a link element that has neither href nor imagesrcset
https://validator.w3.org/nu/

>Error: A link element must have an href or imagesrcset attribute, or both.

![image](https://user-images.githubusercontent.com/34136752/219204029-cd29b8ac-33d5-4d29-ae27-be58fd992b3a.png)
